src/vivarium_gates_mncnh/components/intervention.py

```diff
@@ -508,6 +508,62 @@ def apply_iv_iron_to_gestational_age(
         return exposure
 
 
+class IVIronEffectOnStillbirth(Component):
+    """IV iron effects on stillbirth."""
+
+    @property
+    def columns_required(self) -> list[str]:
+        return [
+            COLUMNS.IV_IRON_INTERVENTION,
+        ]
+
+    #################
+    # Setup methods #
+    #################
+
+    def setup(self, builder: Builder) -> None:
+        builder.value.register_value_modifier(
+            PIPELINES.BIRTH_OUTCOME_PROBABILITIES,
+            self.adjust_stillbirth_probability,
+            requires_columns=self.columns_created,
+        )
+
+    def build_all_lookup_tables(self, builder: Builder) -> None:
+        stillbirth_rrs = builder.data.load(data_keys.IV_IRON.STILLBIRTH_RR)
+        self.lookup_tables["stillbirth_relative_risk"] = self.build_lookup_table(
+            builder=builder,
+            data_source=stillbirth_rrs,
+            value_columns=["value"],
+        )
+
+    ##################################
+    # Pipeline sources and modifiers #
+    ##################################
+
+    def adjust_stillbirth_probability(
```

Comment:
This is the same as oral iron except we subset to those who have IV iron instead of oral iron and use IV iron RRs.